CRM/Iats/Transaction.php

```diff
@@ -448,7 +448,7 @@ static function get_future_monthly_start_dates($start_date, $allow_days) {
         $key = ''; // date('YmdHis');
       }
       else {
-        $display = strftime('%B %e, %Y', $start_date);
+        $display = CRM_Utils_Date::customFormat(date('Y-m-d H:i:s', strtotime($start_date)), '%B %e, %Y');
```

Comment:
start_date is a timestamp, you definitely don't want to run strtotime there.

and let's let civi do all the work ....